src/helpers/getOsLanguage.ts

```diff
@@ -1,29 +1,35 @@
 import { I18nManager, Platform, Settings } from "react-native";
```

Comment:
should we update this file name to `getOsLocale` instead since this is what's being used the most now?